alis-commons.conf

```diff
@@ -16,7 +16,7 @@ PARTITION_BOOT_NUMBER=""
 PARTITION_ROOT_NUMBER=""
 DEVICE_ROOT=""
 DEVICE_LVM=""
-LUKS_DEVICE_NAME="cryptroot"
+LUKS_DEVICE_NAME="root"
```

Comment:
What is the reason for this change?

Reply:
The main reason can be found on https://www.freedesktop.org/software/systemd/man/systemd-gpt-auto-generator.html.

There it is stated that "When systemd is running in the initrd the / partition may be encrypted in LUKS format as well. In this case, a device mapper device is set up under the name /dev/mapper/root".